### PR TITLE
Update dockerfile to centos and update readme

### LINKED ISSUE #787: Upgrade Yarn
### This will require a CR.

When we have a chance we should [upgrade our Yarn version](https://github.com/cfpb/hmda-platform-ui/blob/76e429efbfce9d3fba5b42e35e670b87fad09c9b/Dockerfile#L10). They have released [1.0.0 and beyond](https://github.com/yarnpkg/yarn/releases) with performance gains.

### LINKED ISSUE #798: Switch to Centos-based image
Better mirrors platform auth and removes dev dependency on the debian-based nginx image

### PR DESCRIPTION
**Pull request by wpears**
Closes #798 
Closes #787

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ _All pages, expect the home page, require authentication._
 
 ## Dependencies
 
-We use npm and yarn to manage front-end dependencies. You should have [npm installed](https://nodejs.org/en/) as well as [yarn](https://yarnpkg.com/lang/en/docs/install/).
+We use node and yarn to manage front-end dependencies. You should have [node installed](https://nodejs.org/en/) as well as [yarn](https://yarnpkg.com/lang/en/docs/install/).
```
Comment (awolfe76) [inline README.md:31]: Should we update `package.json` too, to remove `npm` from the scripts?

**Review by awolfe76 — CHANGES_REQUESTED** [review]
Just the one thing about removing `npm` completely. Otherwise looks good.

File: README.md (hunk 1 above)
Comment (wpears) [reply README.md:31]: I'll do that as part of #789

Comment (awolfe76) [reply README.md:31]: 👍

File: Dockerfile [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
+    yum-config-manager --enable cr && \
+    yum update -y && \
+    yum install -y nginx-1.10.2 \
+    yum clean all && \
+    usermod -l $NGINX_USER nginx && \
+    groupmod -n $NGINX_USER nginx && \
+    rm -rf /usr/share/nginx/ && \
+    ls -d -1 /etc/nginx/* | grep -v '\/mime.types$' | xargs rm -rf && \
+    mv nginx/* /etc/nginx && \
+    sed -i 's/80/8080/g' /etc/nginx/nginx.tmpl && \
```
Comment (hkeeler) [inline Dockerfile:19]: Sorry, just now checking this out.  Now that we're running the container on 8080 here, might as well tweak the [`nginx.tmpl`](https://github.com/cfpb/hmda-platform-ui/blob/master/nginx/nginx.tmpl#L41) to also have 8080, then we won't have to do this find/replace here.  :)

Comment (wpears) [reply Dockerfile:19]: Roger

**Pull request merged**